pkg/controller.v1/control/podgroup_control.go

```diff
@@ -125,23 +125,28 @@ var _ PodGroupControlInterface = &VolcanoControl{}
 
 // SchedulerPluginsControl is the  implementation of PodGroupControlInterface with scheduler-plugins.
 type SchedulerPluginsControl struct {
-	Client client.Client
+	Client        client.Client
+	SchedulerName string
 }
 
 func (s *SchedulerPluginsControl) DecoratePodTemplateSpec(pts *corev1.PodTemplateSpec, job metav1.Object, _ string) {
+	if len(pts.Spec.SchedulerName) == 0 && s.GetSchedulerName() != "default-scheduler" {
```

Comment:
why check `GetSchedulerName != "default-schedule"`?

Reply:
if gang-scheduler-name=default-scheduler, skip set a schedulerName in podTemplateSpec.spec.schedulerName.

if gang-scheduler-name is set to a different value(e.g. scheduler-plugins, koordinator), we should set the schedulerName in podTemplateSpec.spec.schedulerName.

Reply:
why not just always set?